types/node/package.json

```diff
@@ -0,0 +1,11 @@
+{
+    "private": true,
+    "types": "index",
+    "typesVersions": {
+        ">=3.1.0-0": {
```

Comment:
should this be ">=3.1"?

Reply:
No. `>=3.1` expands to `>=3.1.0` and excludes prerelease builds. To include those you have to use a full version with a prerelease modifier of `-0`, which includes _all_ 3.1 builds.